server/database/database.js

```diff
@@ -1,6 +1,6 @@
 const mongoose = require('mongoose');
 require('dotenv').config();
-mongoose.connect(process.env.localDB);
```

Comment:
Do you know how these are supposed to work? You can create a file called `.env` in the root folder (next to package.json and such), then assign that same string (without quotation marks) to `localDB`. We'll have to do something different once it's deployed or it will immediately be taken over by bots.